cluster_tol: 0.08       # cluster tolerance which affect resulting cluster size
cluster_min: 10         # cluster minimum number of points
cluster_max: 350        # cluster maximum nubmer of points
reconst_radius: 0.13    # radius of the cluster reconstruction

marker_sx: 0.27         # marker scale x
marker_sy: 0.27         # marker scale y
marker_sz: 0.7          # marker scale z
marker_alpha: 0.4       # alpha value of marker
marker_r: 0.0           # red
marker_g: 1.0           # green
marker_b: 0.0           # blue

lidar_hori_res: 1024    # lidar horizonal resolution
lidar_vert_res: 64      # lidar vertical resolution

filter_factor: 0.5      # filter factor (used for tuning rule based filter)
magic_offset: 9         # temporary measure to align lidar image bounding box

## ------------------------------------
## lidar intensity image classification
## ------------------------------------
clf_img_w: 32           # classifier input image width
clf_img_h: 32           # calssifier input image height
clf_max_batch: 50       # classifier input max batch size

clf_onnx: "/models/lidar_cone_classifier.onnx" # path to classifier model [onnx]
clf_trt: "/models/lidar_cone_classifier.trt"   # path to classifier model [trt]

## ------------------------------------------
## lidar intensity image crop data colleciton
## ------------------------------------------
experiment_no: 6              # experiment number
cone_colour: "y"              # y for yellow, b for blue
save_path: "/tmp/lidar_imgs/" # path where lidar image crops are saved to